ngraph/frontend/paddlepaddle/src/op/elementwise_ops.cpp

```diff
@@ -0,0 +1,86 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <map>
+
+#include <ngraph/opsets/opset6.hpp>
+#include "elementwise_ops.hpp"
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+
+template <typename T>
+OutputVector elementwise_ops (const NodeContext& node) { 
+    auto x = node.get_ng_input("X");
+    auto y = node.get_ng_input("Y");    
+
+    auto axis = node.get_attribute<int>("axis");
+
+    MY_ASSERT(x.get_partial_shape().rank().is_static());
```

Comment:
Please add error message to each assert. User should understand why something failed